cypress/integration/login_admin.js

```diff
@@ -10,15 +10,15 @@ describe.skip('Admin login test', () => {
 
         // should be visible after Log In click
         cy.get('#login-modal').should('not.be.visible');
-        cy.contains('Log In').click();
+        cy.get('.slate-omnibar.mobile-hidden').contains('Log In').click();
```

Comment:
@Camus1859 in the future, it's better to augment the source application with semantic `data-*` attributes for tests to anchor to than to rely on any presentation-centric CSS tags